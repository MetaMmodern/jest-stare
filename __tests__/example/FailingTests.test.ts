describe("Failing Tests", () => {

    // it("should fail with a snapshot difference", () => {
    //     const jsonObject: any = {
    //         name: "pointy",
    //         breed: "beagle",
    //         height: 2,
    //         color: "brown and yellow"
    //     };
    //     expect(JSON.stringify(jsonObject, null, 2)).toMatchSnapshot();
    // });

    it("should be a placeholder", () => {
        expect(true).toBe(true);
    });

    // it("should fail without a snapshot difference", () => {
    //     expect(true).toBe(false);
    // });
});
