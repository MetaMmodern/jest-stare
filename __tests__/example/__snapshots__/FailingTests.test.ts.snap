// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Failing Tests should fail with a snapshot difference 1`] = `
"{
  \\"name\\": \\"pointy\\",
  \\"breed\\": \\"corgi\\",
  \\"height\\": 2,
  \\"color\\": \\"brown and white\\"
}"
`;
